--- shadow-utils.spec.orig	2023-01-18 17:25:08.506459454 -0800
+++ shadow-utils.spec	2023-01-18 17:24:31.586013663 -0800
@@ -4,12 +4,12 @@
 Release: 17%{?dist}
 Epoch: 2
 URL: http://pkg-shadow.alioth.debian.org/
-Source0: https://github.com/shadow-maint/shadow/releases/download/v%{version}/shadow-%{version}.tar.xz
-Source1: https://github.com/shadow-maint/shadow/releases/download/v%{version}/shadow-%{version}.tar.xz.asc
+Source0: shadow-%{version}.tar.xz
+Source1: shadow-%{version}.tar.xz.asc
 Source2: shadow-utils.useradd
 Source3: shadow-utils.login.defs
 Source4: shadow-bsd.txt
-Source5: https://www.gnu.org/licenses/old-licenses/gpl-2.0.txt
+Source5: gpl-2.0.txt
 
 ### Globals ###
 %global includesubiddir %{_includedir}/shadow 
@@ -123,6 +123,8 @@
 for all users. The useradd, userdel, and usermod commands are used for
 managing user accounts. The groupadd, groupdel, and groupmod commands
 are used for managing group accounts.
+This package is rebuilt from the original released SRPM by RCIC @ UC Irvine
+The binaries in rebuilt package do not have calls to sss_cache.
 
 
 ### Subpackages ###
@@ -215,6 +217,7 @@
 
 autoreconf
 %configure \
+        --with-sssd=no \
         --enable-shadowgrp \
         --enable-man \
         --with-audit \
