--- shadow-utils.spec.orig	2024-06-07 16:55:09.023722731 -0700
+++ shadow-utils.spec	2024-06-10 10:49:49.259298404 -0700
@@ -1,17 +1,26 @@
 Summary: Utilities for managing accounts and shadow password files
-Name: shadow-utils
+##RCIC rename shadow-utils --> shadow-utils-no-sssd
+Name: shadow-utils-no-sssd
 Version: 4.6
 Release: 22%{?dist}
 Epoch: 2
 URL: http://pkg-shadow.alioth.debian.org/
-Source0: https://github.com/shadow-maint/shadow/releases/download/v%{version}/shadow-%{version}.tar.xz
-Source1: https://github.com/shadow-maint/shadow/releases/download/v%{version}/shadow-%{version}.tar.xz.asc
+Source0: shadow-%{version}.tar.xz
+Source1: shadow-%{version}.tar.xz.asc
 Source2: shadow-utils.useradd
 Source3: shadow-utils.login.defs
 Source4: shadow-bsd.txt
-Source5: https://www.gnu.org/licenses/old-licenses/gpl-2.0.txt
+Source5: gpl-2.0.txt
 
 ### Globals ###
+## RCIC Changes: prefix,sysconfdir,mandir,includedir redefinition.  turn of build-id debug info
+%define _prefix /opt/RCIC/shadow-utils/%{version}
+%define _sysconfdir %{_prefix}/etc
+%define _mandir %{_prefix}/man
+%define _includedir %{_prefix}/include
+%define _build_id_links none
+## END RCIC
+
 %global includesubiddir %{_includedir}/shadow 
 
 ### Patches ###
@@ -130,6 +139,8 @@
 for all users. The useradd, userdel, and usermod commands are used for
 managing user accounts. The groupadd, groupdel, and groupmod commands
 are used for managing group accounts.
+This package is rebuilt from the original released SRPM by RCIC @ UC Irvine
+The binaries in rebuilt package do not have calls to sss_cache.
 
 
 ### Subpackages ###
@@ -225,7 +236,9 @@
 %endif
 
 autoreconf
+## RCIC CHANGE - add -with-sssd=no to configure
 %configure \
+        --with-sssd=no \
         --enable-shadowgrp \
         --enable-man \
         --with-audit \
@@ -364,6 +377,14 @@
 %{_mandir}/man8/lastlog.8*
 %{_mandir}/man8/vipw.8*
 %{_mandir}/man8/vigr.8*
+## RCIC - Add these for clean removal of the RPM
+%dir %{_sysconfdir}
+%dir %{_bindir}
+%dir %{_sbindir}
+%dir %{_mandir}
+%dir %{_datadir}
+## END RCIC
+
 
 %files subid
 %{_libdir}/libsubid.so.*
